ddp-workspace/projects/ddp-singular/playwright-e2e/.env.singular.test

```diff
@@ -0,0 +1,2 @@
+baseURL=https://singular.test.datadonationplatform.org
+sitePassword=broad_institute
```

Comment:
Should we keep it here? 
what about [the note](https://github.com/broadinstitute/ddp-angular/pull/1759/files#:~:text=%2D%20**DO%20NOT**%20store%20passwords%20in%20%60.env%60%20files.) ?

Reply:
hmm, is this secret?

Reply:
kind of. 
It is a password anyway, even not a critical one.

Reply:
👍🏻 Removed.